content/200-concepts/100-components/02-prisma-client/051-working-with-fields/200-working-with-scalar-lists-arrays.mdx

```diff
@@ -55,9 +55,43 @@ const createdUser = await prisma.user.create({
 })
 ```
 
+## Unsetting the value of a scalar list
+
+<Admonition type="warning">
+
+This method is available on MongoDB only in versions
+[3.11.1](https://github.com/prisma/prisma/releases/tag/3.11.1) and later.
+
+</Admonition>
+
+The following example demonstrates how to [`unset`](/reference/api-reference/prisma-client-reference#unset) <span class="api"></span> the value of a scalar list (`coinflips`):
+
+```ts
+const createdUser = await prisma.user.create({
+  data: {
+    email: 'eloise@prisma.io',
+    coinflips: {
+      unset: true,
+    },
+  },
+})
+```
+
+Unlike `set: null`, `unset` removes the list entirely.
+
 ## Adding items to a scalar list
 
-In [2.20.0](https://github.com/prisma/prisma/releases/2.20.0) and later (PostgreSQL only), you can use the atomic [`push`](/reference/api-reference/prisma-client-reference#push) <span class="api"></span> method to add a single value to a scalar list:
+<Admonition type="warning">
+
+Available for:
+
+- PostgreSQL in versions [2.15.0](https://github.com/prisma/prisma/releases/tag/2.15.0) and later
+- CockroachDB in versions [3.9.0](https://github.com/prisma/prisma/releases/tag/3.9.0) and later
+- MongoDB in versions [3.11.0](https://github.com/prisma/prisma/releases/tag/3.11.0) and later
+
+</Admonition>
+
+Use the atomic [`push`](/reference/api-reference/prisma-client-reference#push) <span class="api"></span> method to add a single value to a scalar list:
```

Comment:
In general I'd be careful to call things atomic, the push itself is atomic, the operation as a whole could be argued to not be atomic - I'd just leave out `atomic`.

Reply:
Thanks, done. Leftover wording from previous version of docs.